DXMainClient/PreStartup.cs

```diff
@@ -278,19 +267,28 @@ private static void CheckPermissions()
                 return;
 
             string error = string.Format(("You seem to be running {0} from a write-protected directory.\n\n" +
-                "For {1} to function properly when run from a write-protected directory, it needs administrative priveleges.\n\n" +
-                "Would you like to restart the client with administrative rights?\n\n" +
-                "Please also make sure that your security software isn't blocking {1}.").L10N("Client:Main:AdminRequiredText"), MainClientConstants.GAME_NAME_LONG, MainClientConstants.GAME_NAME_SHORT);
+                "For {1} to function properly when run from a write-protected directory, it needs administrative priveleges.").L10N("Client:Main:AdminRequiredText1"),
+                MainClientConstants.GAME_NAME_LONG, MainClientConstants.GAME_NAME_SHORT);
+
+            string question = string.Format(("Would you like to restart the client with administrative rights?\n\n" +
+                "Please also make sure that your security software isn't blocking {0}.").L10N("Client:Main:AdminRequiredText2"), MainClientConstants.GAME_NAME_SHORT);
```

Comment:
Is the `security software` part really relevant only for the restart prompt? If not - it should be moved out of that part.

Reply:
Moved this sentence to `error` from `question` string